pyne/alara.py

```diff
@@ -25,8 +25,10 @@
 
 from pyne.mesh import Mesh, MeshError
 from pyne.material import Material, from_atom_frac
-from pyne.nucname import serpent, alara, znum, anum
-from pyne.data import N_A
+from pyne.nucname import serpent, alara, znum, anum, id
```

Comment:
`id()` is a Python builtin function.  Don't import directly.  Do something like:

```python
from pyne import nucname
nucname.id(...)
```